README.md

```diff
@@ -26,7 +28,38 @@ assert_eq!(vec![3.0, 3.5, 4.0, 4.5], items);
 assert_eq!(vec![3.0, 5.0], 3.0.lerp_iter_closed(5.0, 2).collect::<Vec<_>>());
 ```
 
-Of course, the real benefit is that it's derivation is broad enough that it also covers types such as `num::Complex<T>`. If you have an array-processing library, and the arrays are `T: Add<Output = T> + Mul<F: Float, Output = T>`, it'll just work for them as well.
+Of course, the real benefit is that it's derivation is broad enough that it also
+covers types such as `num::Complex<T>`. If you have an array-processing library,
+and the arrays are `T: Add<Output = T> + Mul<F: Float, Output = T>`, it'll just
+work for them as well.
+
+As well as working for individual float values, the crate also provides a derive
+macro, available with the `derive` feature, which will be able to generate an
+implementation automatically. In order to use the derive macro, your structures must
+have only float type fields and all fields much be the same type (all f64 or all f32).
+
+The automatic lerp derivation will simply lerp each of the fields and works on both
+named and tuple type structs.
+
+```toml
+[dependencies]
+lerp = { version = "0.1", features = ["derive"] }
```

Comment:
```suggestion
lerp = { version = "0.4", features = ["derive"] }
```

Might as well bump this now: current version is `0.3`, and adding a feature like this is easily worth another version bump.